### DISCUSSION TITLE
Allow window controls to be clickable (usable) while an alert/message is active

### DISCUSSION THREAD

**Discussion by mateiandrei94** [rustdesk/rustdesk#7588] [General]
### Request

Allow window controls (dragging a window, close, minimize, maximize, etc.) while there is an alert that "grays out" the entire window until an action is taken or the alert goes away automatically.

Currently, the controls are disabled while there is any message / popup.

### Operating system
Microsoft Windows
Version 23H2 (OS Build 22631.3296)

### Rustdesk version
Version: 1.2.3
Build date: 2023-10-13 10:57

### How to reproduce
Somehow find a way to get a message popup while working in a session. I found that it can be easily reproduced like this:
In an active session, disconnect the remote host (it's easier with a virtual machine, simply shut it down), and you will get a message "connection error".
This is normal behavior, however, you will notice that while the alert is displayed, you cannot drag the window around, and the window controls are disabled (not clickable) until you click either OK or reconnect.

### Expected behavior
Regardless of what happens inside an application, I expect to be able to drag the window around, minimize, maximize, close, etc.

### Screenshot
![image](https://github.com/rustdesk/rustdesk/assets/31540942/424d710e-3e70-43f9-a44c-9939165a0da3)

**Comment by rustdesk** [comment]
already in nightly build.
